Cargo.toml

```diff
@@ -27,23 +27,23 @@ args = ["structopt"]
 color-eyre = "0.5"
 educe = "0.4"
 itertools = "0.9"
-openssh = "0.6"
+openssh = "0.7"
```

Comment:
I think you mean `0.8`?

Reply:
Yeah. (Just noticed it and force-pushed, sorry.)